src/tlv.erl

```diff
@@ -400,3 +404,6 @@ unpack_cstring(Tag, <<Tag:?TLV_TAG_SIZE, Len:?TLV_LEN_SIZE, Val/binary>>) ->
 
 unpack_octstring(Tag, <<Tag:?TLV_TAG_SIZE, Len:?TLV_LEN_SIZE, Val/binary>>) ->
     pdu_data:bin_to_octstring(Val, Len).
+
+discard_tag(Tag, <<Tag:?TLV_TAG_SIZE, Len:?TLV_LEN_SIZE, Val/binary>>) ->
+    {undefined, binary:part(Val, Len, byte_size(Val) - Len)}.
```

Comment:
There is no need to calculate `byte_size(Val)`, it's possible just with bit syntax:

```erlang
    <<_:Len/unit:8, Rest/binary>> = Val, 
    {undefined, Rest}.
``` 

It's shorter and probably more optimal way.